src/components/Button/styles.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+export const ButtonStyled = styled.button`
+  width: 179px;
+  height: 36px;
+  width: 100%;
+  height: 100%;
+  border: none;
+  background-color: ${({ backgroundColor }) => backgroundColor};
```

Comment:
Talvez seja bom colocar uma cor default para o botão.

Aconselho utilizar a primária que se repete em varios locais 
```suggestion
  background-color: ${({ backgroundColor }) => backgroundColor || 'cor primaria'};
```

Reply:
Ainda persiste